### DISCUSSION TITLE
Why is the place to display the license the setting screen?

### DISCUSSION THREAD

**Discussion by ishowta** [mono0926/LicensePlist#205] [Q&A]
私は弁護士ではありません。これは法的助言ではありません。

なぜライセンスを表示する場所が設定画面なのでしょうか？
設定とはソフトウェアに対する変更可能なオプションを列挙したものであって、ライセンスの表示は一方的で不変なものであって設定では無いのではないでしょうか。そうであれば、iOSの設定画面が設定の表示及び変更以外の目的を想定したデザインになっていない限り、ライセンスの表示を確認したいユーザーが設定画面を開くことを期待するのは難しいように思えます。
見当違いの質問でしたらすみません。

I am not a lawyer. This is not legal advice.

Why is the place to display the license the setting screen?
A setting is a list of modifiable options for the software, and the license notice is unilateral and immutable, not a setting. Therefore, unless the iOS settings screen is designed for a purpose other than setting, it seems difficult to expect users to open the settings screen to check the license announcement.
Sorry if the question is misleading.

**Comment by mono0926** [comment]
This tool is for developers who want to put the license list on the setting app, so if you don't like it, please use other tools or do it yourself.

This repository is not the place to discuss whether or not it is a good idea to put the license list on the setting app, nor am I personally interested in discussing it.

CocoaPods also has the ability to put licenses on the setting app as well.
[Acknowledgements - CocoaPods/CocoaPods Wiki](https://github.com/CocoaPods/CocoaPods/wiki/Acknowledgements)

[Freeform](https://apps.apple.com/jp/app/id6443742539), the latest app from Apple, also put licenses on the settings app.

**Comment by acevif** [comment]
@ishowta You can use https://github.com/yhirano/LicensePlistViewController instead if you like.

Putting licenses under the settings screens doesn't fit me too, but I can't find other screens to fit.

IMHO, Apple should draw up a design guideline for license screens, as well as manage App's [SBOM](https://ntia.gov/page/software-bill-materials) for advanced security.
